database/schema/controller.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package schema
+
+import "strings"
+
+// ControllerDDL is used to create the controller database schema at bootstrap.
+func ControllerDDL() []string {
+	delta := `
+CREATE TABLE lease_type (
+	id		INT PRIMARY KEY,
+	type	TEXT
+);
+
+INSERT INTO lease_type VALUES
+	(0, 'controller'), 
+	(1, 'model' ), 
+	(2, 'application');
+
+CREATE TABLE IF NOT EXISTS lease (
+	uuid			TEXT PRIMARY KEY,
+	lease_type_id	INT NOT NULL,
+	name			TEXT,
```

Comment:
Do we want to use TABS for alignment here? I know in Github it doesn't align nicely.